### DISCUSSION TITLE
Ironmon Quality of Life

### DISCUSSION THREAD

**Discussion by TRON-7122** [besteon/Ironmon-Tracker#488] [Q&A]
I have been looking to find a quality of life update or install for Kaizo Ironmon (Fire Red). I normally watch someone play who has the hidden items displayed on the ground and they have a nurse joy in the woods to heal so you don't have to talk back all the time o the PC, and there's some quicker dialogue options along with a Pokémon nickname-r in every PC. What quality of life needs to be added in order for me to download and or add it?

**Comment by Fellshadow** [comment]
that's the [Faster FireRed](https://github.com/DrMaple/Faster-FireRed/releases/latest) patch by DrMaple

If you're in the [ironmon discord](https://discord.com/invite/jFPYsZAhjX) then QoL patches like this can be found in the resources channel in there, for future reference
